src/gui/torrentcontentfiltermodel.cpp

```diff
@@ -90,7 +90,7 @@ bool TorrentContentFilterModel::lessThan(const QModelIndex &left, const QModelIn
     TorrentContentModelItem::ItemType rightType = m_model->itemType(m_model->index(right.row(), 0, right.parent()));
 
     if (leftType == rightType)
-      return Utils::String::naturalCompareCaseSensitive(vL, vR);
+      return Utils::String::naturalCompareCaseInsensitive(vL, vR);
```

Comment:
@Chocobo1: Personally, I think it might work best if you make it OS-based.  Operating systems with case-sensitive file management (UNIX, Linux), should use `NaturalCompareCaseSensitive`, while operating systems with case-insensitive file management (Windows, macOS), would have better compatibility with `NaturalCompareCaseInsensitive`.  

At least, I think that would work better.